_posts/2016-03-12-make-your-data-obey.markdown

```diff
@@ -0,0 +1,200 @@
+---
+layout: post
+title: "Make Your Data Obey"
+date: 2016-03-12 10:00:00
+categories: obey data validation modelling
+author: kent.safranski
+cover: /assets/images/covers/obey.jpg
+description: Data is what programming is all about. Applications would be nothing without that beautiful i/o of data moving between modules, services, and data storage. But data is nothing if it's not consistent.
```

Comment:
Consider shortening and adding call to action for sustained organic traffic, e.g.

![image](https://cloud.githubusercontent.com/assets/440298/13732211/f2dc9124-e94a-11e5-9fd6-72c2ebbbd1e9.jpeg)